src/components/Card/NewCard.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { MouseEventHandler, useState } from 'react';
+import styles from './NewCard.module.css';
+
+// components
+import Button from '../UI/Button';
+import Card from '../UI/CardWrapper';
+import DeleteIcon from '../UI/Icons/DeleteIcon';
+import Input from '../UI/Input';
+
+interface NewCardProps {
+  onCancelNewCard: MouseEventHandler;
+  onSaveNewCard: Function;
+}
+
+const NewCard: React.FC<NewCardProps> = (props) => {
+  const [formStep, setFormStep] = useState<number>(1);
+  const [firstStepInputValue, setFirstStepInputValue] = useState<string>('');
+  const [secondStepInputValue, setSecondStepInputValue] = useState<string>('');
+  const cardData = {
+    firstPage: '',
+    secondPage: '',
+  };
+
+  const nextStepHandler = () => {
+    // TODO validation
+    setFormStep(2);
+  };
+
+  const prevStepHandler = () => {
+    setFormStep(1);
+  };
+
+  const inputValueHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (formStep === 1) {
+      setFirstStepInputValue(event.target.value);
+    } else if (formStep === 2) {
+      setSecondStepInputValue(event.target.value);
+    }
+  };
+
+  const saveCard = () => {
+    // TODO validation
+    cardData.firstPage = firstStepInputValue;
+    cardData.secondPage = secondStepInputValue;
+    props.onSaveNewCard(cardData);
+  };
+
+  return (
+    <Card>
+      <div className={styles.content}>
+        {formStep == 2 && (
+          <>
+            <p className={styles['first-side-text']}>{firstStepInputValue}</p>
+            <DeleteIcon
+              className={styles['delete-icon']}
```

Comment:
**nitpick:** It will be easier to navigate through modules when:

```ts
   className={styles.deleteIcon}
```

instead of string literals.

Reply:
I have read in several sources that you should not use camel case in css. That's why it's done this way

Reply:
> I have read in several sources that you should not use camel case in css. That's why it's done this way

Hmm, but don't you have BEM in mind? It's different approach when you work with CSS modules, and plain CSS, because you don't have to worry about scoping. My experience from various companies is that we mostly used camelCase, as it's easier to navigate straight to styles from IDE. 

But as I see around the internet it's more like matter of preference - both approaches are correct. However it's important to be consistent, and stick to one.

https://www.dhiwise.com/post/how-to-structure-and-organize-react-css-modules#:~:text=One%20popular%20naming%20convention%20is%20to%20use%20%27camelCase%27%20for%20class%20names.%20This%20means%20that%20the%20first%20letter%20of%20each%20word%20(except%20for%20the%20first%20word)%20is%20capitalized.%20For%20example%2C%20%27myButton%27%2C%20%27mainContainer%27%2C%20%27headerTitle%27%2C%20etc